golang/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION="1.21"
+ARG VERSION="1.22"
```

Comment:
Could you add 1.22 in this file?
https://github.com/okteto/devenv/blob/main/docker-compose.golang.yml#L12